.github/workflows/build_wheels.yml

```diff
@@ -38,7 +38,7 @@ jobs:
       run: |
         python -m cibuildwheel --output-dir wheelhouse
 
-    - uses: actions/upload-artifact@v3
+    - uses: actions/upload-artifact@v4
       with:
         name: wheels
```

Comment:
Actually I think these need to be unique names like `wheels-${{ strategy.job-index }`